jokes.md

```diff
@@ -0,0 +1,19 @@
+1. A pair of cows were talking in the field. One says, “Have you heard about the mad cow disease that’s going around?”
+
```

Comment:
I think we can delete the spaces between the joke and their "responses" right? It sort of gives the impression that the response is a separate joke on its own.

Reply:
Which spaces? The spaces between the words or the spaces between the sentences?

Reply:
The spaces b/n the sentences. Each joke is in a call-response format right? So we don't need an extra space between the call and response.